app/models/manageiq/providers/workflows/automation_manager/workflow_instance.rb

```diff
@@ -39,7 +39,16 @@ def queue_callback(state, message, _result)
 
   def run
     creds = credentials&.transform_values do |val|
-      ManageIQ::Password.try_decrypt(val)
+      if val.start_with?("$.")
+        name, field = val.match(/^\$\.(?<name>.+)\.(?<field>.+)$/).named_captures.values_at("name", "field")
+
+        authentication = Rbac.filtered_object(ext_management_system.authentications.find_by!(:name => name), :userid => userid, :miq_group_id => miq_group_id)
```

Comment:
NOTE I had to use `miq_group_id` here instead of `miq_group` because we dropped the `OwnershipMixin` from the core PR pending a discussion about how we want to RBAC these

Reply:
can we do a `where` instead of `find_by?`

Where will only bring back the objects once.
find_by will bring back all records, send them back to the db, and then download again.

Reply:
Huh? `find_by` does a `SELECT LIMIT 1` not sure how that brings back all the records

Reply:
Okay done